recipe/meta.yaml

```
@@ -1,39 +1,56 @@
 # https://pagure.io/python-daemon/issue/94
-{% set version = "3.0.1" %}
+{% set name = "python-daemon" %}
+{% set alt_name = name.replace("-", "_") %}
+{% set version = "3.1.2" %}
 
 package:
-  name: python-daemon
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/p/python-daemon/python-daemon-{{ version }}.tar.gz
-  sha256: 6c57452372f7eaff40934a1c03ad1826bf5e793558e87fef49131e6464b4dae5
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ alt_name }}-{{ version }}.tar.gz
+  sha256: f7b04335adc473de877f5117e26d5f1142f4c9f7cd765408f0877757be5afbf4
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
-  skip: True  # [win or (linux and s390x)]
+  skip: True  # [py<37]
 
 requirements:
   host:
     - python
     - pip
-    # Install won't work with setuptools >69.5.1
-    - setuptools <=69.5.1
+    - setuptools >=62.4.0
     - wheel
     - docutils
+    - changelog-chug
   run:
     - python
     - lockfile >=0.10
-    - setuptools
     # docutils doesn't seem necessary here, but pip check won't pass without it.
     - docutils
 
 test:
+  source_files:
+    - test
   requires:
     - pip
+    - pytest
+    - testscenarios # [py<312]
```

https://pagure.io/python-daemon/blob/af81dd5e22642dbd1ffcb6e03425d0877943a12b/f/pyproject.toml#_118
```suggestion
    - testscenarios >=0.4 
```

---

True, that's the test requirement. I still need to skip `py312`, though. Will fix.